apps/calendar/locales/calendar.en-US.properties

```diff
@@ -49,7 +49,7 @@ modify-account-unable-to-sync=Unable to sync
 modify-account-sign-in=Sign in
 
 # presets
-preset-caldav=CalDav
+preset-caldav=CalDAV
```

Comment:
One gotcha of working with strings is that we need to notify the localization folks of string changes. We need to update the ```preset-caldav``` key in this file, and where it's used in order to do so.